csharp/src/ProtocolBuffers.Test/Collections/RepeatedFieldTest.cs

```diff
@@ -239,21 +239,19 @@ public void Clone_ReturnsMutable()
         public void Enumerator()
         {
             var list = new RepeatedField<string> { "first", "second" };
-            using (var enumerator = list.GetEnumerator())
```

Comment:
No, this *should* be okay - and we *should* dispose of iterators. That's what foreach will do, for example. We should look more carefully into what's wrong here...